src/DistributedTracingManager.ts

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { BaseExplorer } from "./baseExplorer";
+import { Constants, DistributedSettingUpdateType } from "./constants";
+import { TelemetryClient } from "./telemetryClient";
+import { Utility } from "./utility";
+import iothub = require("azure-iothub");
```

Comment:
For the sake of consistency, I prefer the `import * as iothub from "azure-iothub";` style since it's used by other code in this project.

Reply:
fixed